src/main/webapp/js/instructorFeedbackEdit/questionRubric.js

```diff
@@ -233,9 +227,7 @@ function highlightRubricRow(index, questionNum, highlight) {
 }
 
 function highlightRubricCol(index, questionNum, highlight) {
-    var idSuffix = getQuestionIdSuffix(questionNum);
-    
-    var $rubricCol = $('.rubricCol' + idSuffix + '-' + index);
+    var $rubricCol = $('.rubricCol-' + questionNum + '-' + index);
```

Comment:
Feels odd having the question number for rubrics be in the middle when other questions have it at the end - maybe open another issue for this?